Weather/Weather/Adapter/WAdapter.h

```diff
@@ -0,0 +1,20 @@
+//
+//  WAdapter.h
+//  Weather
+//
+//  Created by Sangwook Wi on 5/3/17.
+//  Copyright © 2017 Sangwook Wi. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "WModel.h"
+
+@interface WAdapter : NSObject
+
+- (WModel *)convertDataToModel:(NSData *)data;
+- (NSDictionary *)rawDataToDict:(NSData *)data;
+- (NSMutableArray *)dictToArray:(NSDictionary *)dict;
+- (void)updateModelwith:(NSMutableArray *)data of:(WModel *)model;
```

Comment:
The name of the method should be fully camel case `updateModelWith`.
Changing the name to `- (void)updateModelWith:(NSArray *)data model:(WModel *)model` might make it more clear.

Does this method needs to receive a mutable array? It seems like having NSArray as the parameter will give a good fit for what you need